TwitchLib.Api.Core/HttpCallHandlers/TwitchHttpClient.cs

```diff
@@ -98,7 +99,7 @@ public int RequestReturnResponseCode(string url, string method, List<KeyValuePai
                 RequestUri = new Uri(url),
                 Method = new HttpMethod(method)
             };
-            var response = _http.SendAsync(request).GetAwaiter().GetResult();
+            var response = await _http.SendAsync(request);
```

Comment:
just need to add `.ConfigureAwait(false)` to each of the async calls to return to the correct context :)

Reply:
Hope I catched all now